src/main/scala/org/allenai/deep_qa/experiments/datasets/OmnibusDa.scala

```diff
@@ -0,0 +1,70 @@
+package org.allenai.deep_qa.experiments.datasets
+
+import org.json4s._
+import org.json4s.JsonDSL._
+
+/**
+ * This object contains a bunch of JValue specifications for the
+ * Omnibus DA dataset's data files.
+ */
+object omnibusDaDatasets {
```

Comment:
Just call this `OmnibusDa`, to match the filename.  I think this is the right format for these things going forward.